modules/ROOT/pages/shared-resources.adoc

```diff
@@ -17,31 +18,30 @@ To share the metadata, keep the Mule domain project open in Anypoint Studio. Oth
 
 == Prerequisites
 
-This document assumes that you are using Anypoint Studio 7 with Mule runtime 4, or that you are building your apps using Studio 7 and deploying them to Mule 4 Standalone runtime.
+* Anypoint Studio 7.x
+* Mule runtime engine 4.x
 
 == Limitations
 
 Defining flows, subflows, or any message processors as shared resources is not supported. Domains are not meant to share behavior. They are meant to share resources only.
 
-== Basic Anatomy
+== Architecture
 
-=== Studio Project
+A Mule domain Studio project has the following key files:
```

Comment:
Is this Mule domain Studio project the same thing referred to earlier as the Mule domain project/these files?
Should we mention Anypoint Studio in the earlier text?